conferences/presentations/RIghtsCon-2021-06-07/script.md

```diff
@@ -0,0 +1,55 @@
+# Script for lightning talk at RightsCon2021
+
+8-10 minute recorded talk, in the Human Rights-Centered Design track - probably about 1500 words
+(possibly also with an AMA during the conference)
+
+## Original pitch
+([original application](https://github.com/alan-turing-institute/the-turing-way/blob/master/conferences/abstracts/2021-01-RightsCon.md))
+Data scientists make data-driven decisions throughout the lifecycle of their projects. 
+It is their duty to ensure that their work maintains a high level of moral integrity as well as the highest scientific standards, especially, when it impacts people’s lives. 
+In this presentation, we will introduce The Turing Way, an open source book project that involves and supports its diverse community in developing and sharing resources that make data science reproducible, ethical, collaborative and inclusive. 
+In particular, we will demonstrate the Turing Way guide for Ethical Research, the community behind its development, and its human rights-centred approach to data science ethics as a practice that includes activism for change. 
+Most importantly, we will demonstrate how RightsCon attendees can get involved!
+
+(based heavily on [this talk](https://zenodo.org/record/3968454#.YIA1Vmgo-i5) by Malvika Sharan, which was 10 minutes.
+
+## Skeleton:
+1. Intros and what is the Turing Way? (2 mins)
```

Comment:
I think that adding a very short motivation for the talk/the turning way, before jumping into saying what is TTW may help hooking watchers. I will suggest a bit of the script in this direction below